modules/Story/Story.tsx

```diff
@@ -31,47 +30,25 @@ function Story({ story }: Props) {
         return null;
     }
 
-    const { title, subtitle, content, format_version, categories, links } = story;
-    const headerImage = story.header_image ? JSON.parse(story.header_image) : null;
-    const hasHeaderImage = Boolean(headerImage);
+    const { categories, links } = story;
     const hasCategories = categories.length > 0;
+    const nodes = JSON.parse(story.content);
 
     return (
         <Layout>
             <StorySeo story={story} noindex={!isAnalyticsEnabled} />
             <article className={styles.story}>
-                <div
-                    className={classNames(styles.container, {
-                        [styles.withImage]: hasHeaderImage,
-                    })}
-                >
+                <div className={styles.container}>
+                    {isEmbargoStory(story) && <Embargo story={story} />}
```

Comment:
Because header image is now rendered via the `ContentRenderer`, it would require extra work to put Embargo in between the header image and story content (as you would have to extract the image element and render it separately).

After a discussion, we've decided to move the Embargo panel to the top of the story, which also increases its visibility (ignore the expanded image, it's going to be contained in the live version):

![localhost_3000_i-listened-to-animals-hearts-to-reveal-their-hidden-emotional-worlds (1)](https://github.com/prezly/theme-nextjs-bea/assets/4209081/ed6c1c04-c366-454c-92f7-af3aa56ce9f6)